File: Leetcode-Solutions/Arrays/kidsWithGreatestCandy.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function(candies, extraCandies) {
    const maxCandies=Math.max(...candies)
    let result=candies.map(a=>(a+extraCandies)>=maxCandies)
    return result
};

File: Leetcode-Solutions/Arrays/kidsWithGreatestCandy.py
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        """
        :type candies: List[int]
        :type extraCandies: int
        :rtype: List[bool]
        """
        result=[]
        l=max(candies)

        for i in candies:
            maxer=max(i+extraCandies,l)
            result.append(maxer==(i+extraCandies))
        return result
